orchagent/routeorch.cpp

```diff
@@ -1612,19 +1686,28 @@ bool RouteOrch::addRoute(RouteBulkContext& ctx, const NextHopGroupKey &nextHops)
             /* IP neighbor is not yet resolved */
             else
             {
-                if(overlay_nh)
+                if(overlay_nh && !srv6_nh)
```

Comment:
Both overlay_nh and srv6_nh can be true in any case?

Reply:
It can be either overlay_nh or srv6_nh, both cannot be true.